auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -71,7 +71,7 @@
                     </div>
                   </IconTooltip>
                   <IconTooltip
-                    v-if="daysLeft(item.effectiveDate) && daysLeft(item.effectiveDate) <= 3"
+                    v-if="daysLeft(item.futureEffectiveDate) && daysLeft(item.futureEffectiveDate) <= 3"
```

Comment:
This line computes daysLeft twice in some cases. Consider having a single computed instead.

Reply:
Updated